miniwin/minimfc/include/minimfc.h

```diff
@@ -166,7 +166,7 @@ inline BOOL ShowWindow(HWND hWnd, int nCmdShow)
 	return TRUE;
 }
 
-inline BOOL SetWindowPos(HWND hWndInsertAfter, int X, int Y, int cx, int cy, UINT uFlags)
+inline BOOL SetWindowPos(CWinApp** hWndInsertAfter, int X, int Y, int cx, int cy, UINT uFlags)
```

Comment:
[SetWindowPos is a CWnd member function](http://www.icodeguru.com/VC%26MFC/MFCReference/html/_mfc_cwnd.3a3a.setwindowpos.htm)
So this should be:
```c++
BOOL CWnd::SetWindowPos(int X, int Y, int cx, int cy, UINT uFlags)
```

Reply:
I'm not really sure how to implement that or if it requires changes to the decomp project so if you could take care of it it would be much appreciated :)